### ISSUE TITLE
port to neoforge

### ISSUE THREAD

**Issue by Lgmrszd** [LEWeee/pudding#1]
forge bad

**Comment by LEWeee** [comment]
Yeah I wanted to port it either to either neoforge or fabric since I've heard fabric is a lot easier and more intuitive.

I'll probably port it to neoforge some time and then all following updates will be on that version.
